frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/prefill-vm-template-state-update.ts

```diff
@@ -200,7 +199,10 @@ export const prefillVmTemplateUpdater = ({ id, dispatch, getState }: UpdateOptio
           isCloudInitForm = false;
         }
       } else if (volumeWrapper.getType() === VolumeType.DATA_VOLUME && !dataVolume) {
-        const newDataVolumeName = joinIDs(VM_TEMPLATE_NAME_PARAMETER, diskWrapper.getName());
+        const newDataVolumeName = generateDataVolumeName(
+          iGetVmSettingValue(state, id, VMSettingsField.NAME),
```

Comment:
we should use the template name here, because the vm name might not be filled yet when this prefill fires.

Reply:
Done!